ArduPlane/Log.cpp

```diff
@@ -234,7 +234,7 @@ const struct LogStructure Plane::log_structure[] = {
 // @Field: ThrOut: scaled output throttle
 // @Field: RdrOut: scaled output rudder
 // @Field: ThrDem: demanded speed-height-controller throttle
-// @Field: Aspd: airspeed estimate
+// @Field: Aspd: airspeed estimate (measurement if airspeed sensor present and in use)
```

Comment:
I don't think this is actually correct.  I'm reading this as "it is valid if you have an airspeed sensor present and in use".

DCM overrides this to use an actual estimate (rather than a measurement) if we don't have an actual sensor.

Perhaps you meant that to be "(or measurement "?

We should probably remove the word "estimate" from the function names somewhere - something you get from a sensor is really a "measurement" rather than an estimate.  That's probably a different discussion :-)

Reply:
That's what I was trying to say... I think. 
I thought "airspeed measurement (or estimate based on GPS groundspeed and estimated wind if airspeed sensor not in use)" was too verbose.
And I didn't think the current wording was ambiguous, but I'm obviously on the wrong end of the globe :)